File: Practice Codes/1d.java
public class Greeting {
    public static void greet() {
        System.out.println("Hello!");
    }

    public static void greet(String name) {
        System.out.println("Hello, " + name + "!");
    }

    public static void greet(String name, int age) {
        System.out.println("Hello, " + name + "! You are " + age + " years old.");
    }

    public static void main(String[] args) {
        greet();                            // Prints "Hello!"
        greet("John");                      // Prints "Hello, John!"
        greet("Alice", 25);                 // Prints "Hello, Alice! You are 25 years old."
    }
}

File: Practice Codes/1d.swift
func greet() {
    print("Hello!")
}

func greet(name: String) {
    print("Hello, \(name)!")
}

func greet(name: String, age: Int) {
    print("Hello, \(name)! You are \(age) years old.")
}

greet()                     // Prints "Hello!"
greet(name: "John")         // Prints "Hello, John!"
greet(name: "Alice", age: 25)   // Prints "Hello, Alice! You are 25 years old."
